Helpers/VesselHelper.cs

```diff
@@ -74,6 +74,12 @@ public static class VesselHelper {
           if (Module.moduleValues.GetValue ("name") == module) {
             return true;
           }
+		  //if the module being searched for is ModuleDataTransmitter (stock antenna), and that module was not found, now look for ModuleRTAntenna, the RemoteTech Antenna Module.
```

Comment:
if you are making a full alias list, for the ModuleSAS in the satellite
check (which I am guessing is to check for a probe part), RT gives exactly
every probe the module "ModuleSPU" - which is nice, as there are some
probes which lack ModuleSAS.

On Wed, Aug 12, 2015 at 3:51 PM, Aryeh Baruch <abzb13@gmail.com> wrote:

> no problem.
>
> On Wed, Aug 12, 2015 at 3:50 PM, iamchairs <notifications@github.com>
> wrote:
>
>> In Helpers/VesselHelper.cs
>> <https://github.com/iamchairs/StateFunding/pull/4#discussion_r36905205>:
>>
>> > @@ -74,6 +74,12 @@ public static class VesselHelper {
>> >            if (Module.moduleValues.GetValue ("name") == module) {
>> >              return true;
>> >            }
>> > +		  //if the module being searched for is ModuleDataTransmitter (stock antenna), and that module was not found, now look for ModuleRTAntenna, the RemoteTech Antenna Module.
>>
>> Gotcha. Let me clean this up a bit by adding a aliases so we don't end up
>> with hundreds of edge cases and I'll merge this in. Thanks!
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/iamchairs/StateFunding/pull/4/files#r36905205>.
>>
>
>